CONTRIBUTING.md

```diff
@@ -29,14 +29,12 @@ accurate comments, etc.) and any other requirements (such as test coverage).
 
 4.  To run all packages tests simply do `npm test` (we use [jest](https://github.com/facebook/jest)). If you want to work on a specific package and run the tests only for that package, we recommend to use `npm run test:package`. This will prompt you to select one of the available packages. To run in _watch_ mode simply do `npm run test:package -- --watch`.
 
-5.  Integration tests are separated out in another folder "/integration-tests". To run the integration test, you need to create a env file as specified [here](https://commercetools.github.io/nodejs/sdk/api/getCredentials.html). Then run integration test with
+5.  Integration tests are separated out in another folder "/integration-tests". To run the integration test, you need to create an env file that have `npm_config_projectkey` (is equal to projectKey), `PROJECT_KEY`, `CLIENT_ID` and `CLIENT_SECRET` in your environment variables as specified [here](https://commercetools.github.io/nodejs/sdk/api/getCredentials.html). Then run integration test with
```

Comment:
Could you please write an example of this env file? I tried with the following and it did not work.

```
npm_config_projectkey=******************
PROJECT_KEY=******************
CLIENT_ID=******************
CLIENT_SECRET=******************
```

I saved this to a file called `.ct-credentials.env` inside the integration tests folder, like it said in your link here:
https://commercetools.github.io/nodejs/sdk/api/getCredentials.html#from-a-file

Reply:
The npm_config_projectkey, PROJECT_KEY, CLIENT_ID,  and CLIENT_SECRET need to be exported to the process env.

Reply:
That didn't work for either. It then complains like below.

```
ERR: Could not find environment variable CT_***PROJECT-NAME***
```

Could you please try the steps you write yourself first? So you make sure it works on your end first.

Reply:
export `npm_config_projectkey`=`projectkey`, 
then 
export `CT_PROJECTKEY`=`CLIENT_ID`:`CLIENT_SECRET`

Reply:
Ok, please put that into the file. Right now it says

> you need to export `npm_config_projectkey` (is equal to projectKey), `PROJECT_KEY`, `CLIENT_ID` and `CLIENT_SECRET` as environment variables as specified [here](https://commercetools.github.io/nodejs/sdk/api/getCredentials.html) into process env